src/main/java/org/opensearch/sdk/SDKClient.java

```diff
@@ -147,6 +172,17 @@ public OpenSearchClient initializeJavaClient(ExtensionSettings settings) {
         return initializeJavaClient(settings.getOpensearchAddress(), Integer.parseInt(settings.getOpensearchPort()));
     }
 
+    /**
```

Comment:
(Style) Please pair the overloaded client initialize methods together. See https://google.github.io/styleguide/javaguide.html#s3.4.2-ordering-class-contents

Specifically, move this method below `initializeJavaClient()`.